_posts/2022-12-12-deltalake-spark-testing-learnings.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+font: serif
+title: Databricks - An Approach To Unit Testing
+date: 2022-10-27
+highlight: monokai
+author: Bryan Yu
+image: /img/blog/databricks/databricks.png
+tags: databricks deltatable dataframes spark workflows integration unit testing
+description: Lets look at testing our code using delta lake and spark
+---
+
+Before we commence, please note that any examples and discussions below regarding unit testing will be focused on users who are currently writing their code in `python 3` and running orchestrations via `Databricks Workflows` as `Delta Live Tables` are a separate topic altogether. 
```

Comment:
I feel like the intro needs more bang.. why should I read it? Why should I care?

Reply:
mm true, its bit dull